discord/voice_client.py

```diff
@@ -339,18 +331,10 @@ def __init__(self, client: VoiceClient) -> None:
     def ws(self) -> DiscordVoiceWebSocket:
         return self.client.ws
 
-    @property
-    def listening(self) -> bool:
-        return self.is_playing()
-
     def is_listening(self) -> bool:
```

Comment:
`listening` was a property, why isn't `is_listening`?

Reply:
The library usually uses is_x methods for stuff like this. I added these properties in v1.x, but for consistency's sake they aren't necessary here anymore.

Reply:
Interesting, I was referring to the `@property` decorator though